ampd/src/monitoring/endpoints/metrics.rs

```diff
@@ -433,3 +564,22 @@ mod tests {
         result.join("\n") + "\n"
     }
 }
+
+/// System metrics like CPU and memory usage are inherently dynamic and change between test runs
+/// This function normalizes the system metrics to 0 for consistent output in golden file tests.
+#[cfg(test)]
+pub mod test_utils {
+    pub fn normalize_system_metrics(text: &str) -> String {
```

Comment:
```suggestion
    pub fn zeroize_system_metrics(text: &str) -> String {
```
normalization is something else